src/Ombi.Core/Engine/BaseMediaEngine.cs

```diff
@@ -78,6 +78,26 @@ public int ResultLimit
             return _dbTv;
         }
 
+        protected async Task<RequestEngineResult> CheckCanManageRequest(BaseRequest request) {
+            
+            // Admins can always manage requests
+            var isAdmin = await IsInRole(OmbiRoles.PowerUser) || await IsInRole(OmbiRoles.Admin);
+            if (isAdmin)
+                return new RequestEngineResult { Result = true };
+
+            // Users with 'ManageOwnRequests' can only manage their own requests
+            var isRequestedBySameUser = ( await GetUser() ).Equals(request.RequestedUser);
```

Comment:
`( await GetUser() ).Equals(request.RequestedUser);`
An object comparison might not be a good idea here as the object is coming from the request and the `GetUser` might differ slightly. So just comparing the `Id` property would be enough

Reply:
Don't think we did this change? 😀

Reply:
Oops, I did but then I undid it. Stay tuned.